src/promptflow/promptflow/batch/_result.py

```diff
@@ -138,6 +138,7 @@ class BatchResult:
     metrics: Mapping[str, str]
     system_metrics: SystemMetrics
     error_summary: ErrorSummary
+    aggr_error_summary: Mapping[str, Any]
```

Comment:
How about adding `aggr_error_list` in `ErrorSummary`?
Not sure if it's a good idea to have 2 error related summaries here.

Reply:
The current `error_summary` points more to the line error summary, I have 2 proposals:
1. `line_error_summary` and `aggr_error_summary` are in the first level of the batch result.
2. Move `aggr_error_summary` to `error_summary` and refactor `error_summary` to 
![image](https://github.com/microsoft/promptflow/assets/111329184/5da50a39-3e86-4843-a824-75504edf3720)

@huaiyan @thy09 which one do you prefer?

Reply:
Offline synced, we will use proposal 2.